django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
@PeterJCLaw are you trying to convince me to change more things in this repo? I'll only do it, you know. :)

(I wouldn't have made that choice of default backend, personally, but I think changing that would break backwards compatibility)